src/applications/vaos/services/appointment/transformers.js

```diff
@@ -39,6 +55,18 @@ export function getAppointmentType(appt, useFeSourceOfTruthCC) {
   if (appt?.kind !== 'cc' && appt?.requestedPeriods?.length) {
     return APPOINTMENT_TYPES.request;
   }
+
+  // In an upcoming iteration, we will be update the FE source of truth for VA requests as well
+  // eslint-disable-next-line sonarjs/no-collapsible-if
+  if (useFeSourceOfTruthVA) {
+    if (appt?.type === 'VA') {
+      return APPOINTMENT_TYPES.vaAppointment;
+    }
+    // We must return a value for the function, but this is technically only possible when the type is invalid.
+    // We can potentially throw an error here but that's rather unusual in our codebase.
```

Comment:
This looks a little odd so I'll add a bit more explanation. 

The existing logic defaults to the type `APPOINTMENT_TYPES.vaAppointment` when all previous conditions do not apply. In the future, we will probably rename all the values of the `APPOINTMENT_TYPES` to match the "type" field so this entire `getAppointmentType` function won't be needed and we can just check `appt?.type` directly. 

However, in the meantime, Javascript requires us to return some value here when `appt?.type !== 'VA'`. This should never actually occur and is definitely an error scenario. I've elected to simply return the type value itself since it more closely aligns with potential future behavior but we can choose to throw an error here if we want. 

Thoughts?